homeassistant/components/arcticspa/light.py

```diff
@@ -0,0 +1,56 @@
+"""Arctic Spa Light."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pyarcticspas import LightState
+from pyarcticspas.error import SpaHTTPException
+
+from homeassistant.components.light import LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import ArcticSpaEntity, async_refresh_after
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Arctic Spa Light from a config entry."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    async_add_entities([SpaLight(coordinator)])
+
+
+class SpaLight(ArcticSpaEntity, LightEntity):
+    """Represents the Arctic Spa light."""
+
+    _attr_name = "Light"
```

Comment:
I am wondering, will we support more platforms in the future? Or are we just controlling the lights and that is it?

Reply:
No, I've started with lights as the simplest (and least intrusive to the device) integration.

I'll need switches for pumps and other features that can be turned on/off.

I'll need sensors for the pH detection sensor data.